runtests.py

```diff
@@ -161,7 +158,7 @@ def main() -> None:
             "Run the given tests. If given no arguments, run everything except"
             + " pytest-extra and mypyc-extra. Unrecognized arguments will be"
             + " interpreted as individual test names / substring expressions"
-            + " (or, if they end in .test, individual test files)"
+            + " (which can end in .test, to specify individual test files)"
             + " and this script will try to run them."
```

Comment:
Might be easier to just say it's passed onto pytest?

Reply:
That would be easier, but I wouldn't regard it as explanatory enough. I'd like to briefly explain to the user of runtests.py the important info for them to know, like this; instead of just, say, pointing them to https://docs.pytest.org/en/stable/how-to/usage.html#select-tests (which is correct information about what happens).

Figuring out this specific implication is a service we can provide to the reader.